gym/dev_wrappers/torch_wrapper.py

```diff
@@ -17,6 +22,10 @@ def __init__(self, env: Union[Wrapper, Env], device: Optional[torch.device] = No
         for an underlying Jax environment then convert the return observations from Jax arrays
         back to torch tensors.
 
+        Functionality for converting between torch and jax types originally copied from
+        https://github.com/google/brax/blob/9d6b7ced2a13da0d074b5e9fbd3aad8311e26997/brax/io/torch.py
+        Under the Apache 2.0 license. Copyright is held by the authors
+
```

Comment:
I changed the text of the copyright statement to represent that the file wasn't copied but the functionality in the conversion methods specifically. I think this is where you intended for the statement to go?

Reply:
See comment above